src/pages/aclmanagement.html

```diff
@@ -251,7 +260,7 @@ <h1 id="acl-management-view-title">
                                             </button>
                                             <button ng-click="saveRule(activeTabScope)"
                                                     class="btn btn-link save-rule-btn"
-                                                    ng-disabled="!ruleData.$valid"
+                                                    ng-disabled="ruleData.role.$error.required || ruleData.role.$error.minlength"
```

Comment:
Wll, this is anot point for my doubts about altering the ngModel with this new validation. Here you are forced to do this in order to maintain the behavior in the template although it has nothing to do with the new feature